assets/stylesheets/shared/_colors.scss

```diff
@@ -19,6 +19,7 @@ $gray:                   #87a6bc;
 // See wordpress.com/design-handbook/colors/ for more info.
 
 $gray-light:             lighten( $gray, 33% ); //#f3f6f8
+$gray-super-light:       #fbfcfd;
```

Comment:
Heh, wow that is super light, @adambbecker can we just use white (it's so close). Or define a variable in the component CSS with `lighten( $gray, 34% )` or something like that. Would rather not add it to the global.